sum_square_difference30.py

```diff
@@ -0,0 +1,23 @@
+# The sum of the squares of the first ten natural numbers is, 1² + 2² + 3²...+ 10² = 385
+
+# The square of the sum of the first ten natural numbers is, (1+2+3...+10)²=55²= 3025
+
+# Hence the difference between the sum of the squares of the first ten natural numbers and the square of the sum is 
+# 3025 - 385 = 2640.
+
+# Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum
+def First_Ten():
+    first = 0
+    tots = 0
+    res = 0
+    for x in range(101):
+        a = (x ** 2)
+        first = first + a
+        sp = (first)
```

Comment:
what is sp? also you reference sp in line 21, but you the scope of your variable naming is not accurate